src/client.rs

```diff
@@ -19,12 +19,12 @@ impl RustemonClient {
     ///
     /// * `cache_mode` - The [CacheMode] to inject into the client.
     /// * `options` - The [CacheOptions] to inject into the client.
-    pub fn new(cache_mode: CacheMode, options: Option<CacheOptions>) -> Self {
+    pub fn new(cache_path: String, cache_mode: CacheMode, options: Option<CacheOptions>) -> Self {
         Self {
             client: ClientBuilder::new(Client::new())
                 .with(Cache(HttpCache {
                     mode: cache_mode,
-                    manager: CACacheManager::default(),
+                    manager: CACacheManager { path: cache_path },
```

Comment:
Unfortunately, CACacheManager isn’t performing any checks with the path.
With that in mind, would you consider leaving the constructor as is and instead create another new method ? Such as `new_path_unchecked` for exemple ?

Reply:
I've added a new constructor method as requested, let me know if you'd like any further changes, thank you for your consideration!